client/coral-admin/src/routes/Community/components/Community.js

```diff
@@ -74,9 +74,11 @@ export default class Community extends Component {
       );
     }
 
+    // console.log('Community Container', this.props);
+
     return (
       <div>
-        <FlaggedAccounts />
+        <FlaggedAccounts {...this.props} />
```

Comment:
Better be explicit and only pass required props. This makes the interface of FlaggedAccounts vague and couples Community props with FlaggedAccounts. Also bad for optimizations.